client/van_serviceinterface_create_test.go

```diff
@@ -193,43 +205,47 @@ func TestVanServiceInterfaceCreate(t *testing.T) {
 		}
 
 		// Create a router.
-		err = cli.VanRouterCreate(ctx, types.VanSiteConfig{
-			Spec: types.VanSiteConfigSpec{
-				SkupperName:       "skupper",
-				IsEdge:            false,
-				EnableController:  true,
-				EnableServiceSync: true,
-				EnableConsole:     false,
-				AuthMode:          "",
-				User:              "",
-				Password:          "",
-				ClusterLocal:      true,
-			},
-		})
-		assert.Check(t, err, "Unable to create VAN router")
+		if testcase.init {
+			err = cli.VanRouterCreate(ctx, types.VanSiteConfig{
+				Spec: types.VanSiteConfigSpec{
+					SkupperName:       testcase.namespace,
+					IsEdge:            false,
+					EnableController:  true,
+					EnableServiceSync: true,
+					EnableConsole:     false,
+					AuthMode:          "",
+					User:              "",
+					Password:          "",
+					ClusterLocal:      true,
+				},
+			})
+			assert.Check(t, err, "Unable to create VAN router")
+		}
 
 		// Create the VAN Service Interface.
 		service := types.ServiceInterface{
 			Address:  testcase.addr,
 			Protocol: testcase.proto,
 			Port:     testcase.port,
 		}
-		err = cli.VanServiceInterfaceCreate(ctx, &service)
+		observedError := cli.VanServiceInterfaceCreate(ctx, &service)
```

Comment:
I would have expected that when you run against a real cluster, there will be a service created for each service interface created named testcase.addr. It may be that we need a delay to account for the informer latency to discover the services before the next loop on test cases. Delay maybe a second or two and see what happens. Longer term, we will need something more deterministic.